EdurecaInClassProject/src/app/components/search-bus/search-bus.component.ts

```diff
@@ -103,7 +98,9 @@ export class SearchBusComponent implements OnInit {
     else if (this._busService.getBuses().length == 0){
       this.errorMessage = "No buses found for this route";
```

Comment:
we can use translate here also.
translate.instant() method might be a help to you.